omnibus/config/software/chef-dk.rb

```diff
@@ -75,7 +75,7 @@
   appbundle "chef", lockdir: project_dir, gem: "chef", without: %w{integration docgen maintenance ci travis}, env: env
```

Comment:
you need to appbundle `gem: "chef-bin"` instead of "chef" here.

Reply:
(actually it should be in addition to, we still need `gem: "chef"` for knife)

Reply:
We're not pulling in chef 15 yet